samples/wifi/softap/Kconfig

```diff
@@ -77,4 +77,12 @@ config SOFTAP_SAMPLE_PASSWORD
 	string "Passphrase (WPA2) or password (WPA3)"
 	help
 	  Set the password for SoftAP.
+
+config SOFTAP_SAMPLE_MAX_STATIONS
+	int "Maximum number of stations"
+	range 1 1
```

Comment:
Add a comment as to why the special range here: `Experimental support for one, but in future support for up to 8 (nRF70 limitation)`